src/ontology/mondo-edit.obo

```diff
@@ -363905,7 +363905,7 @@ xref: ICD10:K00.4 {source="Orphanet:83450", source="ORDO:83450/ntbt"}
 xref: MESH:D018126 {source="MONDO:equivalentTo"}
 xref: Orphanet:83450 {source="MONDO:equivalentTo"}
 xref: SCTID:66063001 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000001 {source="MONDO:0015603-obsoleted"} ! disease or disorder
+is_a: MONDO:0005172 {source="MONDO:0015603-obsoleted"} ! skeletal system disease
```

Comment:
for the source, you should link to the GitHub ticket and add your ORCID

Reply:
this can have a more specific parent. In Orphanet, they classify this under Rare odontal or periodontal disorder.

We have 'periodontal disease' in Mondo, I suggest this should be the parent.